shared/idmapset_linux.go

```diff
@@ -63,21 +63,21 @@ func (e *IdmapEntry) HostidsIntersect(i IdmapEntry) bool {
 func (e *IdmapEntry) Intersects(i IdmapEntry) bool {
 	if (e.Isuid && i.Isuid) || (e.Isgid && i.Isgid) {
 		switch {
-		case is_between(e.Hostid, i.Hostid, i.Hostid+i.Maprange):
+		case is_between(e.Hostid, i.Hostid, i.Hostid+i.Maprange-1):
```

Comment:
If we do this, I think we can get rid of the +1s in findIdmap since this implies the ranges aren't inclusive.